md6/tests/mod.rs

```diff
@@ -0,0 +1,363 @@
+#![no_std]
```

Comment:
`#![no_std]` is not needed in tests since tests rely on `std` by default.

Could you also provide a provenance link for these test vectors?